src/mixins/createParentMapMixin.ts

```diff
@@ -80,10 +80,13 @@ const createParentMapMixin: ParentMapMixinFactory = compose<ParentMap<Child>, Pa
 		set children(this: ParentMapMixin<Child> & { invalidate?(): void; }, value: Map<string, Child>) {
 			if (!value.equals(childrenMap.get(this))) {
 				value.forEach((widget) => {
-					if (widget.parent !== this) {
-						widget.parent = this;
-						/* TODO: If a child gets attached and reattached it may own multiple handles */
-						getRemoveHandle(this, widget);
+					// Workaround for https://github.com/facebook/immutable-js/pull/919
+					if (widget) {
```

Comment:
It works, but I've tried to keep the workaround separate from the original code.